src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -1731,13 +1731,15 @@ public ValidateSetAttribute(Type valuesGeneratorType)
     /// <summary>
     /// Allows dynamically generate set of values for <see cref="ValidateSetAttribute"/>
     /// </summary>
+#nullable enable
```

Comment:
@iSazonov What is our code convention for indent of pre-processor directives. Visual Studio 2019 is automatically removing any indent here.

Reply:
Related issue: https://github.com/dotnet/roslyn/issues/12306

Reply:
If we can not to turn off the behavior in VS 2019 I agree to put this at the beginning of the line.